docs/releases/1.10.txt

```diff
@@ -350,6 +350,10 @@ This prevents confusion about an assignment resulting in an implicit save.
   :class:`~django.contrib.gis.geos.Point` are deprecated in favor of the
   ``tuple`` property.
 
+* The :attr:`django.contrib.gis.geos.MultiPolygon.cascaded_union` property is
```

Comment:
We usually don't usually include a link to the deprecated method since that we'd have to remove that link when removing the deprecated feature. Don't forget a mention in the deprecation timeline too.